### DISCUSSION TITLE
Connection refused after oauth2 on android

### DISCUSSION THREAD

**Discussion by itsferasinc** [ionic-team/capacitor#3502] [Q&A]
nuxt spa + capacitor. 
After nuxt auth module redirects on other site (inner browser, allowNavigation set) with oAuth2 and permissions granted by 'token' type flow http://localhost (redirect url back to app) opens with error connection refused.  
url example: `http://localhost/login#access_token=eoLT2Z6vF1qUkrfhDAbq3kbnp54glj&expires_in=36000&token_type=Bearer&scope=basic+student&state=G7d01wDsNj16L83a3HfYl`
But if i copy and open same url inside app it will open properly
`
"server": {
		"allowNavigation": [
			"*.testoauthserver.ru"
		]
	}
`

**Comment by tharrington** [comment]
I am having this issue as well
